packages/slice-machine/components/DeleteDocumentsDrawer/DeleteDocumentsDrawerOverLimit.tsx

```diff
@@ -91,7 +99,8 @@ const DeleteDocumentsDrawerOverLimit: React.FunctionComponent = () => {
               label="Try again"
               variant="primary"
               onClick={() => {
-                console.log("Pushing");
+                closeModals();
+                pushChanges(); // TODO: should this confirm deletion? Otherwise we get the other modal
```

Comment:
I'm glad you asked the question because it's an important one that I struggled myself with too 😄.

Should the frontend or the backend control what's being done.
- the frontend can be out of date so it would push wrong things (ex, when someone else push changes, you wouldn't see it straight away so the changes might not be the same and things will fail)
- the goal of pushing things is to push your file system as is in the database. By doing it in the backend, we ensure that this actually happens as much as possible.

There is more to say, it's an interesting question, i'll keep some for the meeting this afternoon.

Reply:
Review: ask Côme about allowing users to select what to push.

Reply:
resolving this, Côme answered that it's not a priority for now.